test/functional/browser.study.api.js

```diff
@@ -56,63 +56,64 @@ function merge(...sources) {
   return Object.assign({}, ...sources);
 }
 
-/** return a studySetup, shallow merged from overrides
- *
- * @return {object} mergedStudySetup
- */
-function studySetupForTests(...overrides) {
-  // Minimal configuration to pass schema validation
-  const studySetup = {
-    activeExperimentName: "shield-utils-test-addon@shield.mozilla.org",
-    studyType: "shield",
-    endings: {
-      ineligible: {
-        baseUrls: [
-          "https://qsurvey.mozilla.com/s3/Shield-Study-Example-Survey/?reason=ineligible",
-        ],
+const publicApiTests = function(studyType) {
```

Comment:
Why is this `const publicApiTests = function(...)` instead of `function publicApiTests(...)`?

Reply:
Arbitrary. Changing to conform to the style of the rest of the script